### ISSUE TITLE
Query about input dimensions

### ISSUE THREAD

**Issue by mm04926412** [scikit-tda/pervect#3]
Is it required that each persistance diagram have the same number of points / the same length?

**Comment by lmcinnes** [comment]
No, they can be of varying size, hence the fact that the input should be a *list* of arrays, rather than a 3d-array.
